node/nodeRunner.go

```diff
@@ -141,6 +143,12 @@ func (nr *nodeRunner) startShufflingProcessLoop(
 			return err
 		}
 
+		log.Debug("creating disabled API services")
+		httpServerWrapper, closableHttpServer, err := nr.createDisabledHttpServer()
```

Comment:
createInitialHttpServer ? Disabled makes me thing of something that has a 0 functionality. Initial functionality has 0.01 functionality :D

Reply:
agree. renamed